api/controllers/deploy_test.go

```diff
@@ -120,7 +121,7 @@ func TestListDeploys(t *testing.T) {
 	}
 
 	mockDeployProvider.EXPECT().
-		List().
+		List(all).
```

Comment:
You need to add this query param to the fireball.Context (see task logs test) and test when all=true

Reply:
:white_check_mark: 

I made this adjustment and added a new test, but I've added a note in the test that the unit test can't actually test properly filtering out Active/Inactive Task Definitions and would have to be done through a system test. As it stands, we're just testing the controller's ability to handle all=false and all=true.